### PR TITLE
Ridho/cashier password

### PR DESCRIPTION
**Pull request by ridhomard**
Test cases should cover: 

Cashier password form validation:

- field is required 
- Minimum field
- password combination check 
- Confirm password check 
- Password check againt current password 

Update cashier password 
Cashiers page check while password is locked 
Unlock the cashier
Main Menu Tab file 
Cashier Page file

### CODE REVIEW

File: src/test/java/testCases/Validate_CashierPassword.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        CashierPassword_Action.checkTitle(driver);
+    }
+    //Check empty field validation
+    @Test(priority=3)
+    public void check_emptyField() {
+        CashierPassword_Page.cashierPassword(driver).clear();
+        CashierPassword_Page.cashierPassword(driver).sendKeys("");
+        CashierPassword_Page.confirmPassword(driver).clear();
+        CashierPassword_Page.confirmPassword(driver).sendKeys("");
+        CashierPassword_Page.updateButton(driver).click();
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_CashierPassword.java]: Any specific reason for not using CashierPassword_Action methods to enter into fields and submitting form in each of the tests ?

File: src/test/java/testCases/Validate_CashierPassword.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        CashierPassword_Page.cashierPassword(driver).clear();
+        CashierPassword_Page.cashierPassword(driver).sendKeys("a");
+        CashierPassword_Page.updateButton(driver).click();
+        Assert.assertEquals(CashierPassword_Page.errMsg_1(driver).getText(),"You should enter 6-25 characters.");  
+    }
+    //Check validation for invalid cashier password 
+    @Test(priority=5)
+    public void check_invPass() {
+        CashierPassword_Page.cashierPassword(driver).clear();
+        CashierPassword_Page.cashierPassword(driver).sendKeys("sssssssss");
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_CashierPassword.java]: Please avoid static values and use the RandomStringUtils methods to generate any random values

**Review by bijan2007 — COMMENTED** [review]
Please look at my comments above

File: src/test/java/testCases/Validate_CashierPassword.java (hunk 1 above)
Comment (ridhomard) [reply src/test/java/testCases/Validate_CashierPassword.java]: Hi @bijan2007  i am sorry ,, i just followed your method in https://github.com/binary-com/binary-static-ci/blob/master/src/test/java/testCases/Validate_PersonalDetailsPage.java . 

I also think it should be done all in CashierPassword_Action . It is best if we can standardise it! Let me change this .

File: src/test/java/testCases/Validate_CashierPassword.java (hunk 2 above)
Comment (ridhomard) [reply src/test/java/testCases/Validate_CashierPassword.java]: Sure i will change this.

**Review by bijan2007 — APPROVED** [review]

**Pull request merged**